### ISSUE TITLE
incrémentation HP et HC

### ISSUE THREAD

**Issue by yann44** [mickeys27/conso#8]
Bonjour,
j'ai installé le plugin suivi Conso pour utiliser les valeurs de mon ZW95 1 pince.
J'ai bien fait l'agenda HP et HC, sur le graphe consommation du jour il y a bien une période bleue et rouge qui correspondent à mes HC et HP.
Par contre, dans le tableau consommation, les HC et HP sont incréments de la même façon.

![capture d ecran 2019-02-11 a 10 15 20](https://user-images.githubusercontent.com/26790806/52587979-ace2be80-2e3b-11e9-8ef2-9bb3a1989e65.png)

Comment puis-je faire pour y remédier?

Merci d'avance!

**Comment by mickeys27** [comment]
Salut
Tu dois vérifier tes commandes dans la configuration de ton équipement.
A mon avis tu as mis la même chose.

**Comment by yann44** [comment]
C'est tout à fait ça, j'ai mis la même source venant du ZW95 au lieu du virtuel HP et HC....

Je vais changer ça et voir si ça marche....
Merci!

**Comment by yann44** [comment]
effectivement, c'était ça l'erreur, j'ai changé et ça incrémente maintenant les bonnes valeurs!

Deuxième petit soucis, comment on rentre les informations d'index HC et HP initiaux?

![capture d ecran 2019-02-12 a 07 56 06](https://user-images.githubusercontent.com/26790806/52617405-febb3100-2e9b-11e9-9c0c-900fbaaed4bb.png)

J'ai 54561kWh en HC et 87870kWh en HP

Bonne journée!

**Comment by mickeys27** [comment]
Regarde dans l' onglet outil la il y a les boutons de synchronisation

**Comment by yann44** [comment]
celui la?

![capture d ecran 2019-02-12 a 12 26 29](https://user-images.githubusercontent.com/26790806/52632356-95e6af80-2ec1-11e9-927a-e3e46b347fb0.png)

**Comment by mickeys27** [comment]
Oui en bas

**Comment by yann44** [comment]
Désolé de faire mon boulet, mais c'est où?

![capture d ecran 2019-02-12 a 14 33 00](https://user-images.githubusercontent.com/26790806/52638861-3396aa80-2ed3-11e9-82e3-dc9669861b98.png)
